src/live_restore/live_restore_fs.c

```diff
@@ -864,8 +889,8 @@ __wti_live_restore_fs_restore_file(WT_FILE_HANDLE *fh, WT_SESSION *wt_session)
         if ((time_diff_ms / (WT_THOUSAND * WT_PROGRESS_MSG_PERIOD)) > msg_count) {
             __wt_verbose(session, WT_VERB_LIVE_RESTORE_PROGRESS,
               "Live restore running on %s for %" PRIu64
-              " seconds. Currently copying offset %" PRId64 " of size %" WT_SIZET_FMT,
-              lr_fh->iface.name, time_diff_ms / WT_THOUSAND, read_offset, lr_fh->source_size);
+              " seconds. Currently copying offset %" PRId64 " of size %" PRId64,
+              lr_fh->iface.name, time_diff_ms / WT_THOUSAND, read_offset, WTI_BITMAP_END(lr_fh));
```

Comment:
`WTI_BITMAP_END(lr_fh)` looks wrong here. Why would we background migrate a file in `size(bitmap)` chunks?

Reply:
The size(bitmap) is the portion of the file tracked by the bitmap.